pallet/Cargo.toml

```diff
@@ -24,7 +24,6 @@ pallet-timestamp = { default-features = false, git = "https://github.com/parityt
 sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", rev = "8bbad73f5" }
 sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", rev = "8bbad73f5" }
 sp-staking = { default-features = false, git = "https://github.com/paritytech/substrate.git", rev = "8bbad73f5" }
-frame-election-provider-support = { default-features = false, git = "https://github.com/paritytech/substrate.git", rev = "8bbad73f5" }
```

Comment:
Should be possible to get rid of staking-pallet related dev-dependencies now.

Reply:
Right, removed.